xwiki-platform-core/xwiki-platform-blog/xwiki-platform-blog-ui/src/main/resources/Blog/Categories.xml

```diff
@@ -116,7 +116,7 @@ $xwiki.jsx.use('Blog.ManageCategories')##
       </name>
     </class>
     <property>
-      <description/>
+      <description> </description>
```

Comment:
Looks like `mvn xar:format` has not been executed here. Moreover, this change is meaningless regarding the issue.